scripts/newinstall.sh

```diff
@@ -34,23 +38,27 @@ LSST_HOME="$PWD"
 cont_flag=false
 batch_flag=false
 help_flag=false
+noop_flag=false
 
 # Use system python to bootstrap unless otherwise specified
 PYTHON="${PYTHON:-/usr/bin/python}"
 
-while getopts cbhP: optflag; do
+while getopts cbhnP: optflag; do
 	case $optflag in
-	c)
-		cont_flag=true
-		;;
-	b)
-		batch_flag=true
-		;;
-	h)
-		help_flag=true
-		;;
-	P)
-		PYTHON=$OPTARG
+		c)
```

Comment:
I can live with this indenting scheme for a 'case', but I'm not sure it adds to readability (at the expense of an extra indent level).